src/completion.rs

```diff
@@ -1519,109 +1468,13 @@ impl From<BuiltinType> for VarType {
     }
 }
 
-fn create_completion_package_removed(
```

Comment:
I'm also not familiar with the `_removed` functions, would it make sense to ask the original author if they're around?

Reply:
Since we do not have any tests that indicate that we lose any feature I'd be inclined to just ignore it and just move forwards. There are enough gaps in the LSP feature-wise anyway that we ought not worry about some potential (minor) loss of functionality.